app/src/main/res/layout/recycler_row_message_sent.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Layout for any message sent by the current user-->
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/message_layout"
```

Comment:
we have two message_layout? but different xmls?

Reply:
two message layouts for different types of messages. it was easier to name them the same thing than to make an if or switch statement

Reply:
think of it as an implicit interface.

Reply:
I'm not sure if we can have two different components with the same id, but as long as it works on your end, then it's good.